oxidation/parsec_cli/parsec_sequestre_dump/src/model.rs

```diff
@@ -0,0 +1,47 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use parsec_api_crypto::SecretKey;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Queryable, Serialize, Deserialize, PartialEq)]
+pub struct Block {
+    pub block_id: Vec<u8>,
+    pub data: Vec<u8>,
+    pub author: i32,
+    pub size: i32,
+    pub created_on: f32,
```

Comment:
Sqlite use String to store Timestamp ^^

Reply:
yup we store the data in sqlite as float (as btw this should be a f64, see https://github.com/Scille/parsec-cloud/pull/2363#discussion_r882473832), but we want to manipulate in the code a datetime object.
So we need to put the conversion somewhere, and it would be best to have it here ;-)